src/components/Banner/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -85,6 +85,14 @@ exports[`<Banner /> renders correctly when open 1`] = `
   height: 24px;
 }
 
+@media screen and (min-width:480px) {
+
+}
+
+@media screen and (min-width:480px) {
+
+}
```

Comment:
Here we see this empty media query because of this mixin: `responsiveSizeMixin`.
This mixin expects to receive `size` object in parameters.
And according to this parameter, it should draw some styled with media queries.

`
export const responsiveSizeMixin = size =>
  size
    ? css`
        font-size: ${typography.size[
            typeof size === "string" ? size : size.xSmall
            ]};

        ${size.small &&
        smallAndUp`
    font-size: ${typography.size[size.small]};
  `};

        ${size.medium &&
        mediumAndUp`
    font-size: ${typography.size[size.medium]};
  `};

        ${size.large &&
        largeAndUp`
    font-size: ${typography.size[size.large]};
  `};

        ${size.xLarge &&
        xLargeAndUp`
    font-size: ${typography.size[size.xLarge]};
  `};
      `
        : "";
};
`

But, the `TextBase` component sends `size` props as a string `hecto` instead of an object: 
`{
    xSmall: responsiveSize.xSmall || responsiveSize.small || size,
    small: responsiveSize.small,
    medium: responsiveSize.medium,
    large: responsiveSize.large,
    xLarge: responsiveSize.xLarge
}`

As a result `responsiveSizeMixin` consider string `size.small` as a function. And tries to execute it. But the next string returns undefined because it tries to extract some font size parameter from `typography.size[size.small]`

Reply:
Fixed.